multiqc/modules/gatk/base_recalibrator.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+""" MultiQC submodule to parse output from GATK BaseRecalibrator """
+
+import logging
+from collections import OrderedDict, defaultdict
+from multiqc.plots import bargraph, table, linegraph
+
+# Initialise the logger
+log = logging.getLogger(__name__)
+
+
+class BaseRecalibratorMixin():
+    def parse_gatk_base_recalibrator(self):
+        """ Find GATK BaseRecalibrator logs and parse their data """
+
+        self.gatk_base_recalibrator = dict()
+        for f in self.find_log_files('gatk/base_recalibrator', filehandles=True):
+            parsed_data = self.parse_report(
+                f['f'],
+                {
+                    '#:GATKTable:Arguments:Recalibration argument collection values used in this run': 'arguments',
+                    '#:GATKTable:Quantized:Quality quantization map': 'quality_quantization_map',
+                    '#:GATKTable:RecalTable0:': 'recal_table_0',
+                    '#:GATKTable:RecalTable1:': 'recal_table_1',
+                    '#:GATKTable:RecalTable2:': 'recal_table_2',
+                }
+            )
+            if len(parsed_data) > 0:
+                if f['s_name'] in self.gatk_base_recalibrator:
+                    log.debug("Duplicate sample name found! Overwriting: {}".format(f['s_name']))
+                self.add_data_source(f, section='base_recalibrator')
+                self.gatk_base_recalibrator[f['s_name']] = parsed_data
```

Comment:
As discussed, would be good to have multiple data structures here to avoid having the multi-level `parsed_data`